dmscripts/models/queries.py

```diff
@@ -64,3 +64,10 @@ def add_counts(join, group_by, model, data, directory):
         )['_count'].fillna(0)
 
     return data
+
+
+def expand_json_subfields(field, subfields, data):
```

Comment:
I don't think flatten is the right word - it suggests you're actually moving them, rather than adding them to the parent and leaving them where they are.